ghost/core/core/server/adapters/email/mailgun/index.js

```
@@ -1,47 +1,43 @@
+/* eslint-disable max-lines */
+// Disable max-lines: Email adapter contains cohesive send logic that shouldn't be split
+const EmailProviderBase = require('../EmailProviderBase');
 const logging = require('@tryghost/logging');
```

ESLint

[error] 'logging' is assigned a value but never used.

(no-unused-vars)